changelog/6484.misc.md

```diff
@@ -0,0 +1,19 @@
+## Summary
+- Renamed `templates` to `responses` across Rasa code base and documentation.
+- Renamed `response_templates` to `responses` across the Rasa code base.
+- Renamed `template_name` and `response_name` to `utter_action` in Rasa Core and rasa/shared.
+- Renamed `utter_templates` to `responses` in rasa/shared.
+
+### NLG Server
+- Changed request format to send `response` instead of `template` as a field.
```

Comment:
let's tell them that the old field is still supported until 3.0.0.

Reply:
All deprecation items should go into a `6484.removal.md` changelog. Should've said that yesterday but I'd also add them (copy-paste is enough) to the migration guide.